harmony_browse_image_generator/browse.py

```diff
@@ -286,7 +287,7 @@ def get_color_map_from_image(image: Image) -> dict:
 
     """
     color_tuples = np.array(image.getpalette(rawmode='RGBA')).reshape(-1, 4)
-    color_map = {}
+    color_map = all_black_color_map()
```

Comment:
> Will you ever get a time when color_tuples is significantly less than 256, and you have multiple black values at the end of the colour map? Does that matter?

Yes, we have gotten that before and will still see that. It does not matter.

> Will you ever get a time when colour_tuples is more than 256? Would that be a problem? (It seems like the code will all still work, but maybe nodata would not be set as expected?)

No, because we only ask for 254 values [when quantizing ](https://github.com/nasa/harmony-browse-image-generator/blob/mhs/DAS-2180/update-libraries/harmony_browse_image_generator/browse.py#L256-L262). 

I just tried with 300 for giggles and quantize won't go above 256: `ValueError: bad number of colors`


Also No Data is set right before reprojection, so it would end up overwriting the last color in the table.  Which is also why I only ask for 254 values, so that we have a transparent and no data index available without ruining any of the quantized values.

Reply:
> Possibly nitpicky thought (feel free to ignore), you could use `enumerate` below maybe:
> 
> ```
> for idx, color_tuple in enumerate(color_tuples):
>     color_map[idx] = tuple(color_tuple)
> ```

Sure that's cleaner.